ui/opening/css/_next.scss

```diff
@@ -76,10 +73,5 @@
         text-align: right;
       }
     }
-    &:hover {
-      &__popularity span {
-        background: $c-primary;
-      }
-    }
```

Comment:
`opening__next:hover__popularity span` is supposed to be `opening__next__popularity:hover span`, and also the element is`$c-primary` without hover.